app/assets/javascripts/mpa/counts/ECCountTable.ts

```diff
@@ -0,0 +1,10 @@
+import {CountTable, Count} from './CountTable';
+import {ECOntology} from '../ontology/ec/ECOntology';
+import {Ontologies} from '../ontology/Ontologies';
+
+export class ECCountTable extends CountTable<ECOntology, string>
```

Comment:
Do we need separate classes for each type of `CountTable`? It seems to me that we can simply use `CountTable` in all cases, or is there a specific reason why these are split up?

Reply:
Currently each CountTable implements GetOntology() and the TaxaCountTable has a restricted constructor. If we decide to directly use the corresponding global Ontologies instead of getting them from the CountTables we can indeed get rid of these implementations.